src/DynamoCoreWpf/Views/Core/WorkspaceView.xaml

```diff
@@ -479,7 +479,6 @@
                                     <Popup x:Name="PART_Popup"
                                            AllowsTransparency="true"
                                            Focusable="false"
-                                           HorizontalOffset="-1"
```

Comment:
@QilongTang This was the one causing issue, so we can remove the margin altogether.